<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabsWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1161</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>911</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="box"/>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>20</y>
        <width>191</width>
        <height>58</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="LoadGerberFile">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Generate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSlider" name="ZoomIn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="sliderPosition">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>460</y>
        <width>191</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGraphicsView" name="canvas"/>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>20</y>
        <width>20</width>
        <height>571</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QGraphicsView" name="canvas_2">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>88</y>
        <width>189</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>961</x>
        <y>99</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addPosButton">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>140</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>120</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="x_text">
      <property name="geometry">
       <rect>
        <x>963</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>965</x>
        <y>181</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>X cord</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>1052</x>
        <y>178</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Y cord</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="y_text">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>200</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="isZero">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>240</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>zero</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="isHomography">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>360</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>homography point</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>1052</x>
        <y>388</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Y cord</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="x_text_homo">
      <property name="geometry">
       <rect>
        <x>963</x>
        <y>410</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>965</x>
        <y>391</y>
        <width>70</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>X cord</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="y_text_homo">
      <property name="geometry">
       <rect>
        <x>1050</x>
        <y>410</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>340</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="feeder_text">
      <property name="geometry">
       <rect>
        <x>963</x>
        <y>299</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>965</x>
        <y>280</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Feeder Type (id)</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pointer">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>600</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="FeederTable">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>821</width>
        <height>471</height>
       </rect>
      </property>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Feeder type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Code</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Label</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Skip</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="ImportButton">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>20</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Import</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveButton">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>60</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ReloadButton">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>100</y>
        <width>181</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reload</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QTableWidget" name="pos_table">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>831</width>
        <height>591</height>
       </rect>
      </property>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Feeder Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>X Pos</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y Pos</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IsZero</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IsHomography</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>X Pos Hom.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Y Pos Hom.</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGerber_Viewer_v0_1">
    <property name="title">
     <string>Gerber Viewer v0.1</string>
    </property>
   </widget>
   <addaction name="menuGerber_Viewer_v0_1"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
